elated-egrets/bangazon-internal-site-2.0
# J rview user

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRview_user
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. load the page
1. register a new user and create the profile
1. once the user is created check and make sure the profile is viewable when the tab is pressed.

## What you changed
created the template to view the profile
## What was added to read me
N/A
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
